nextjs-13/src/components/NewsletterItem/index.tsx

```diff
@@ -38,14 +43,24 @@ const NewsletterItem = ({ item, getData }: Props) => {
   return (
     <li className="newsletter-item" data-testid="newsletter-item">
       <span className="newsletter-name">{item.name}</span>
-      <Image
-        src={`/images/icons/trash.svg`}
-        alt={'Delete button'}
-        width={16}
-        height={16}
-        onClick={handleDelete}
-        data-testid="btn-delete"
-      />
+      <div className="icons">
+        <Image
+          src={`/images/icons/edit.svg`}
+          alt={'Delete button'}
+          width={16}
+          height={16}
+          onClick={handleUpdate}
+          data-testid="btn-edit"
+        />
+        <Image
+          src={`/images/icons/trash.svg`}
+          alt={'Delete button'}
+          width={16}
+          height={16}
+          onClick={handleDelete}
+          data-testid="btn-delete"
+        />
```

Comment:
These should be buttons, not images, to be semantically correct.

Reply:
Resolved [here](https://github.com/nimblehq/js-framework-benchmark/pull/118/commits/ce0ea642aa5f42cfb7218eeb1e920c4d7f0431d3)